docs/docs/install/config-file.md

```diff
@@ -129,6 +129,16 @@ The default configuration looks like this:
     "scan": {
       "enabled": true,
       "cronExpression": "0 0 * * *"
+    },
+    "watch": {
+      "enabled": false,
+      "usePolling": true,
+      "interval": 10000,
+      "binaryInterval": 10000,
+      "awaitWriteFinish": {
+        "stabilityThreshold": 2000,
+        "pollInterval": 100
```

Comment:
Right, we can remove the write finish parameters. I think we should keep the polling interval configurable. The default is 10 seconds which I've upped from the chokidar default of 100ms. This has important performance implications so it could be important to configure.

Reply:
Done